modules/support-knowledgebase-search.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * support/support-redhat-knowledgebase.adoc
+
+[id="support-knowledgebase-search_{context}"]
+= Searching the Red Hat Knowledgebase
+
+In the event of an {product-title} issue, perform an initial search to determine if a solution already exists within the Red Hat Knowledgebase.
+
+.Prerequisites
+
+* A Red Hat Customer Portal account.
+
+.Procedure
+
+. Login to the link:http://access.redhat.com[Red Hat Customer Portal].
```

Comment:
s/Login/Log in/
ISG: 
**log in to** _v_
Write as shown - three words. Use only in documentation for UNIX systems, such as AIX. Otherwise, use
“log on to.”

FWIW, I think "log in to" works just as well, even when not documenting UNIX.

Reply:
Resolved.